zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumCnxManager.java

```diff
@@ -1090,7 +1090,7 @@ private void acceptConnections() {
                             break;
                         }
 
-                        LOG.error("Exception while listening", e);
+                        LOG.error("Exception while listening on port {}", address, e);
 
```

Comment:
-  `LOG.error("Exception while listening to address{}", address, e);` is better?
- I will re-test this patch tomorrow

Reply:
I made the change to the error message. Please check now.